tools/coq_makefile.ml

```diff
@@ -800,6 +806,21 @@ let check_overlapping_include (_,inc_i,inc_r) =
 	    Printf.eprintf "Warning: in options -R/-Q, %s and %s overlap\n" pdir pdir') l;
   in aux (inc_i@inc_r)
 
+(* Generate a .merlin file that references the standard library and
+ * any -I included paths.
+ *)
+let merlin targets (ml_inc,_,_) =
+  print ".merlin:\n";
+  print "\t@echo 'FLG -rectypes' > .merlin\n" ;
+  List.iter (fun c ->
+      print "\t@echo \"B $(COQLIB)" ; print c ; print "\" >> .merlin\n")
+    lib_dirs ;
+  List.iter (fun (_,c) ->
+      print "\t@echo \"B " ; print c ; print "\" >> .merlin\n" ;
+      print "\t@echo \"S " ; print c ; print "\" >> .merlin\n")
```

Comment:
You could use `printf "\t@echo \"B %s\" >> .merlin\n" c` for readability (same thing for other sequenced `print` calls.